### DISCUSSION TITLE
Search with and or

### DISCUSSION THREAD

**Discussion by StuartMcD** [solspace/craft-freeform#673] [Questions & Answers]
Im using Freeform as a pseudo membership system and it works well. Now I'm looking at a situation where I need to search 2 fields for one search term. Currently I'm calling 2 x `craft.freeform.submissions({})` calls, 1st pass uses 

`fieldSearch:{
	c_phone: contactID
}` 

Record matches to resultVar

`fieldSearch:{
	s_phone: contactID
}`
and then push results to resultVar and work on it from there. Ideally I'd use the craft default 

```
ie. c_phone:contactID OR s_phone:contactID
{% set searchQuery = searchArray|join(' OR ') %}

{% set comms = craft.entries()
	.section('comms')
	.orderBy('postDate DESC')
	.search(searchQuery)
%}
```

The docs are pretty light here so I'm leaning towards - its not possible but need to ask before moving on, thx.

**Comment by kjmartens** [comment]
Sorry for the delay @StuartMcD,

In speaking with a developer, this likely isn't possible at this time. We can check into what's needed to get it working though and hopefully squeeze in a change soon. Which version of Freeform are you using? 🙂

**Reply by StuartMcD** [reply]
I thought as much, currently on 4.0.22. I'll be watching the changelogs 😉
